.travis/install_openssl_1_1_1.sh

```diff
@@ -30,7 +30,7 @@ INSTALL_DIR=$2
 PLATFORM=$3
 
 cd "$BUILD_DIR"
-curl --retry 3 -L https://github.com/openssl/openssl/archive/OpenSSL_1_1_1-stable.zip --output OpenSSL_1_1_1-stable.zip
+curl --retry 3 -L https://github.com/openssl/openssl/archive/OpenSSL_1_1_1f.zip --output OpenSSL_1_1_1-stable.zip
```

Comment:
Why is the output still being called `OpenSSL_1_1_1-stable.zip`?  I find this misleading as it would map to the `OpenSSL_1_1_1-stable` branch. May be use the term newest release/tagged version instead?

Reply:
The output label can be anything, but it is confusing- updated.